src/main/java/com/amazon/asksdk/cloudfoody/OpenTable.java

```diff
@@ -0,0 +1,42 @@
+package com.amazon.asksdk.cloudfoody;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.*;
+import java.net.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class OpenTable {
+
+    private static final String BASE_URL = "http://opentable.herokuapp.com/api/";
+
+    /*
```

Comment:
NP: Please change the comment formatting, because of the convention.

E.g. 
/**
 *
 */

Reply:
Done, please let me know if I understood this right